test/test_meta_init.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import torch
+from torch.nn.utils.meta_init import (
+    clear_meta_init_cache,
+    is_meta_init,
+    materialize,
+    meta_init,
+)
+from torch.testing._internal.common_utils import TestCase, run_tests
+
+
+class MetaInitTest(TestCase):
+    def test_is_meta_init_returns_correct_value(self) -> None:
```

Comment:
Don't we need a lot more tests here? We should probably have tests covering a large number of common tensor constructor ops.

Reply:
There will be a lot more tests after the materialization PR.